Server/Processors/ServerMessageProcessor.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Domain.Game;
+using Domain.Messages;
+using System;
+using System.Collections.Generic;
+
+namespace Server.Processors
+{
+    public class ServerMessageProcessor : IMessageProcessor
+    {
+        public List<Message> ProcessConnectedMessage(Message message)
+        {
+            var result = new List<Message>();
+
+            var connectionMessage = (ConnectionMessage)message;
+            Lobby.SetPlayerName(connectionMessage.PlayerId, connectionMessage.Name);
+
+            return result;
+        }
+
+        public List<Message> ProcessCreateGameMessage(Message message)
+        {
+            var result = new List<Message>();
+            var createGameMessage = (CreateGameMessage)message;
+
+            var player = Lobby.GetPlayer(message.PlayerId);
+            var game = new Game(player);
+            Lobby.AddGame(game);
+
+            createGameMessage.GameId = game.Id;
+            result.Add(createGameMessage);
+
+            return result;
+        }
+
+        public List<Message> ProcessGetGamesMessage(Message message)
```

Comment:
Method expects GetGamesMessage type so it makes sense to change method signature to List<Message> ProcessGetGamesMessage(GetGamesMessage message) (in IMessageProcessor too)